os-eap-legacy/os-eap7-launch/added/launch/ha.sh

```diff
@@ -2,10 +2,10 @@ source ${JBOSS_HOME}/bin/launch/openshift-node-name.sh
 source $JBOSS_HOME/bin/launch/logging.sh
 
 function prepareEnv() {
-  unset OPENSHIFT_KUBE_PING_NAMESPACE
-  unset OPENSHIFT_KUBE_PING_LABELS
-  unset OPENSHIFT_DNS_PING_SERVICE_NAME
-  unset OPENSHIFT_DNS_PING_SERVICE_PORT
+  #unset OPENSHIFT_KUBE_PING_NAMESPACE
```

Comment:
I don`t think there is a need to update this file, it is not handled by our scripts.

Reply:
ok